strings/en_US/messages.po

```diff
@@ -4243,3 +4243,9 @@ msgstr "In the next year"
 
 msgid "Indicate your first session"
 msgstr "Indicate your first session"
+
+msgid "See all {badgesAmount} badges"
+msgstr "See all {badgesAmount} badges"
+
+msgid "{name} doesn\'t have any badges yet. Talk to the organisers of your Dojo to learn how {name} can be rewarded through badges."
```

Comment:
No need to escape the `'` since the whole string is in `"`s. You need to keep the escape in the Vue template because the string is wrapped in `'`s, but not here. They actually won't manage to match up because this will take the string as if the backslash character is actually there.